org.testeditor.web.backend.persistence/src/test/java/org/testeditor/web/backend/persistence/AbstractPersistenceTest.xtend

```diff
@@ -0,0 +1,61 @@
+package org.testeditor.web.backend.persistence
+
+import com.google.inject.Guice
+import com.google.inject.Injector
+import com.google.inject.Module
+import com.google.inject.util.Modules
+import de.xtendutils.junit.AssertionHelper
+import java.util.List
+import javax.inject.Inject
+import org.junit.Before
+import org.mockito.MockitoAnnotations
+import org.testeditor.web.dropwizard.auth.User
+
+abstract class AbstractPersistenceTest {
```

Comment:
class comment that explains that injection module is prepared here to allow for user injection